src/main/webapp/populateData.js

```diff
@@ -311,3 +312,60 @@ function populateGo() {
       });
 }
 
+/**
+ * Set up the assign panel. For now, this just prints the response
+ * from the server for /gmail-actionable-emails
+ */
+function setUpAssign() {
+  const assignContent = document.querySelector('#assign');
+
+  const subjectLinePhrases = ['Action Required', 'Action Requested'];
+  fetchActionableEmails(subjectLinePhrases, true, 7)
+      .then((response) => {
+        assignContent.innerText = response
+            .map((obj) => obj.snippet)
+            .reduce((a, b) => a + '\n' + b);
```

Comment:
Map and reduce makes your code look clean - I should probably use them more!

Reply:
They're a lot of fun to use once you get the hang of them!

Reply:
Wasn't a fan of .stream() but it's starting to grow on me not gonna lie